test/python/WMCore_t/WMBS_t/JobSplitting_t/FileBased_t.py

```diff
@@ -129,7 +129,7 @@ def createLargeFileBlock(self):
         """
         testFileset = Fileset(name = "TestFilesetX")
         testFileset.create()
-        for i in range(5000):
+        for _ in range(5000):
```

Comment:
I wonder where 5000 comes from. but if it doesn't have to be 5000, whether we can reduce to 1000 if that helps reducing testing time significantly